tests/datasets/test_ogbgraphdataset.py

```diff
@@ -0,0 +1,8 @@
+from gammagl.datasets.ogb_graph import OgbGraphDataset
+
+def test():
+    data=OgbGraphDataset('ogbg-molhiv')
+    print(data)
+    print(data[0])
+
+test()
```

Comment:
1. Replace the function name "test" with "test_ogbgraphdataset", which makes it easier to distinguish the functions from other files.
2. Using "assert" to check the correctness of some variable. e.g. the "feature.shape[0]" and "num_nodes" should be equal.